src/GridFieldOrderableRows.php

```diff
@@ -2,8 +2,8 @@
 
```

Comment:
Looks like we can actually remove this and use the symbiote orderablerows that comes out of the box.

Can delete this place and update the namespace in `PageExtension.php` to `use Symbiote\GridFieldExtensions\GridFieldOrderableRows;`